x/erc20/keeper/token_pairs.go

```diff
@@ -45,7 +46,8 @@ func (k Keeper) GetTokenPairID(ctx sdk.Context, token string) []byte {
 	return k.GetDenomMap(ctx, token)
 }
 
-// GetTokenPair - get registered token pair from the identifier
+// GetTokenPair - get registered token pair from the identifier.
+// Returns 'true' if pair exists, otherwise 'false'.
```

Comment:
the returning bool is a convention for getters if an instance has been `found`, so this comment is not necessary